### PR TITLE
Status card configurable

### PR DESCRIPTION
**Pull request by upatricck**
Did paired codding with @rawjazz and made the status card last_contact configurable

Closing issue #253

### CODE REVIEW

**Comment by upatricck** [comment]
@jlark  Needing a review

File: seshdash/views.py [hunk 1]
```diff
@@ -695,24 +695,30 @@ def get_latest_bom_data(request):
...
+    for measurement, point in latest_points.items():
+        latest_point_data.append({"item":get_measurement_verbose_name(measurement),
+                                  "value":str(round(latest_points[measurement]['value'], 2))
+                                          + get_measurement_unit(measurement)
+                         })
+
+    if 'last_contact' in measurement_list:
+        # adding data from the rmc_status
+        try:
+            # TODO letest_point should only return one point
```
Comment (jlark) [inline seshdash/views.py]: Why not use the RMC_Status model item.

Comment (upatricck) [inline seshdash/views.py]: We it thought should be the same time.

**Comment by rawjazz** [comment]
we thought it would double querrying for the same result
because we already have the time on the point

File: seshdash/views.py (hunk 1 above)
Comment (upatricck) [inline seshdash/views.py]: @jlark Do you think its ok, so that we can merge it???

**Comment by jlark** [comment]
I think it's better to use the RMC last contact value as this should be the single place that this data is gathered from. I think doing it differently in seperate places is harder to test and can cause bugs.

**Comment by upatricck** [comment]
Yeah working on it.

**Pull request merged**